pom.xml

```diff
@@ -129,6 +129,8 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
+		<project.build.outputTimestamp>2024-01-01T00:00:00Z</project.build.outputTimestamp>
```

Comment:
Using a static value seems difficult to manage. I suggest looking at using the git commit timestamp. This would be automatic, it would change with ever commit and ensure reproducible builds get the same timestamp when rebuilt from any commit.

Reply:
Maven-release-plugin updates this field during release .....

Reply:
Yes, the maven-release-plugin should update this field automatically during a release. We do need to verify this at the next release though.